README.adoc

```diff
@@ -89,26 +89,24 @@ If you want to build everything, use the `build` task:
 	$ ./gradlew build
 ----
 
-
-
 == Modules
-There are a number of modules in Spring Boot, here is a quick overview:
+There are several modules in Spring Boot; here is a quick overview:
```

Comment:
The Spring Style Guide (https://github.com/spring-io/spring-doc-resources/blob/master/spring-style-guide/spring-style-guide.adoc) calls for not using semicolons. Make two sentences.